SUASSystem/SUASSystem/sda.py

```diff
@@ -2,44 +2,94 @@
 import multiprocessing
 from SUASSystem.suas_logging import log
 from time import sleep
+import traceback
+import numpy
 
-def run_sda_process(logger_queue, waypoints, sda_status, sda_avoid_coords, vehicle_state_data, mission_information_data):
-    SUASSystem.logging.logger_worker_configurer(logger_queue)
+def run_sda_process(logger_queue, waypoints, sda_status, sda_avoid_coords, UAV_status, vehicle_state_data, mission_information_data):
+    SUASSystem.suas_logging.logger_worker_configurer(logger_queue)
     logger_name = multiprocessing.current_process().name
-
+    '''boundary_points = [
+            {
+                "boundary_pts" : [
+                    {"latitude" : 38.1424887, "longitude" : -76.4344168, "order" : 0},
+                    {"latitude" : 38.1430962, "longitude" : -76.4248037, "order" : 1},
+                    {"latitude" : 38.1517193, "longitude" : -76.4261985, "order" : 2},
+                    {"latitude" : 38.1513987, "longitude" : -76.4351463, "order" : 3}
+                ]
+            }
+        ]'''
     log(logger_name, "Instantiating SDA converter")
     while True:
         try:
+            print('tries to instantiate')
+            #sda_converter = SUASSystem.SDAConverter(vehicle_state_data[0].get_location(), 5)#mission_information_data[0]["fly_zones"])
             sda_converter = SUASSystem.SDAConverter(vehicle_state_data[0].get_location(), mission_information_data[0]["fly_zones"])
             break
         except:
```

Comment:
Ideally, you shouldn't use try-catch-alls, but if you have no other workaround, then that's fine.